test/src/e2e_vm_tests/test_programs/should_pass/conditional_compilation/src/main.sw

```diff
@@ -0,0 +1,102 @@
+script;
+
+configurable {
+  #[cfg(target = "fuel")]
+  CFG_VALUE: u64 = 40,
+  #[cfg(target = "evm")]
+  CFG_VALUE: () = (),
+  #[cfg(target = "fuel")]
+  #[cfg(target = "evm")]
+  CFG_VALUE: () = (),
+}
+
+#[cfg(target = "fuel")]
+const VALUE: u64 = 40;
+#[cfg(target = "evm")]
+const VALUE: () = ();
+#[cfg(target = "fuel")]
+#[cfg(target = "evm")]
+const VALUE: () = ();
```

Comment:
What is the desired behavior when both targets are specified?

Reply:
They are AND'd so the attributed never compiles. Added comments in the test to clear it.